recipes/libiconv/all/conanfile.py

```diff
@@ -5,7 +5,7 @@
 import os
 import functools
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.49.0"
```

Comment:
CI runs 1.47.0, going to that

```suggestion
required_conan_version = ">=1.47.0"
```